index.html

```diff
@@ -109,18 +109,17 @@ <h2>
 
         <pre class="example" title="Example did:web DID document">
 {
-  "@context": "https://w3id.org/did/v1",
+  "@context": "https://www.w3.org/ns/did/v1",
   "id": "did:web:example.com",
-  "publicKey": [{
-       "id": "did:web:example.com#owner",
-       "type": "Secp256k1VerificationKey2018",
-       "owner": "did:web:example.com",
-       "ethereumAddress": "0xb9c5714089478a327f09197987f16f9e5d936e8a"
+  "verificationMethod": [{
+     "id": "did:web:example.com#owner",
+     "type": "Secp256k1VerificationKey2018",
+     "owner": "did:web:example.com",
+     "ethereumAddress": "0xb9c5714089478a327f09197987f16f9e5d936e8a"
```

Comment:
please remove this example, and include examples from:

https://github.com/w3c/did-test-suite/blob/main/packages/did-core-test-server/suites/implementations/did-key-2020-db.json#L42

Reply:
since the proposed context change makes this invalid json-ld.

Reply:
@OR13 ah, right, thanks!